x-pack/solutions/security/plugins/security_solution/common/entity_analytics/utils.ts

```diff
@@ -4,15 +4,9 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import type { ExperimentalFeatures } from '../experimental_features';
-import { EntityType } from './types';
 
-export const getAllEntityTypes = (): EntityType[] => Object.values(EntityType);
+import { EntityType } from './types';
 
-export const getDisabledEntityTypes = (
```

Comment:
As discussed on the last Entity Store sync, we agreed to enable generic entity store only when it is enabled in Kibana. I think tha means it will bring back this logic? The difference is that now instead of using `experimentalFeatures` as Feature Flag, we are now using the `uiSettingsClient` as it's a Kibana advanced setting.

Let me know if you think it's better to handle that in this PR or separately, I can help with setting up the necessary services to have the Kibana setting checking as we have [here](https://github.com/elastic/kibana/blob/5402d2b90ebcd99c3a687544b471846c6c93335a/x-pack/solutions/security/plugins/security_solution/server/lib/asset_inventory/asset_inventory_data_client.ts#L169)